mitodl/open-discussions
# Remove text widget class and add wysiwyg widget field editor

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #1639 

#### What's this PR do?
Adds a `markdown_wysiwyg` field type and updates the markdown widget to use it for the editor.

#### How should this be manually tested?
Create a text widget. Run migrations and verify that the widget instance is deleted.

Edit a markdown widget. You should be able to do anything the editor allows for and it should update the widget markdown accordingly.
